dockstore-webservice/src/main/java/io/dockstore/webservice/resources/TokenResource.java

```diff
@@ -116,7 +116,7 @@
     private static final String BITBUCKET_URL = "https://bitbucket.org/";
     private static final String GITLAB_URL = "https://gitlab.com/";
     private static final String ORCID_URL = "https://orcid.org/";
-    private static final TOSVersion CURRENT_TOS_VERSION = TOSVersion.TOS_VERSION_1;
+    private static final TOSVersion CURRENT_TOS_VERSION = TOSVersion.TOS_VERSION_2;
     private static final PrivacyPolicyVersion CURRENT_PRIVACY_POLICY_VERSION = PrivacyPolicyVersion.PRIVACY_POLICY_VERSION_2_5;
```

Comment:
@NatalieEO How does this work again? I thought I saw a draft of the TOS that includes a privacy policy... Does this need to be updated too?

Reply:
Yeah when I wrote this, I wrote it under the assumption that they would always be separate documents...
Right now, if either are out of date, then the UI will logout the user when they visit Dockstore. Could change the UI to no longer check for the privacy policy if they're going to be together from now.

Reply:
The new TOS references a separate Dockstore privacy document, so I expect they are not combined together. @denis-yuen can you confirm?

Reply:
You're right... I saw "Dockstorte Privacy Statement" during some Zoom meeting when somebody was showing the TOS, and thought that was a privacy statement, but now I see the details are a link to another privacy document. So you can ignore this whole thread. Sorry for the confusion.